examples/repository_deployment_policy/README.md

```diff
@@ -0,0 +1,18 @@
+# Non-Organization Owner Repository Deployment Policy Example
+
+This displays repository deployment policy for non-organization GitHub accounts.
+
+This example will create a repository in the specified `owner` organization. See https://www.terraform.io/docs/providers/github/index.html for details on configuring [`providers.tf`](./providers.tf) accordingly.
```

Comment:
> for non-organization GitHub accounts
> in the specified `owner` organization

How do these two reconcile?

Reply:
This was just a quick way to get it up & running to make sure I can test how it works so I didn't pay much attention to the comments, I think I can delete the whole folder here if that's ok with you

Reply:
Yep, no worries, please go for it.

Reply:
Removed